strapi/strapi
# [Audit logs] Add service

### What does it do?

- Creates a dedicated service for audit logs that subscribes to event hub events
- The service is only enabled for gold license
- It currently just logs to the console

### Why is it needed?

- To prepare to save audit logs via a provider

### How to test it?

- Run the tests: `yarn test:unit packages/core/strapi/lib/services/__tests__/audit-logs-service.test.js`
- Perform actions in Strapi, like editing content for example


Expect the following log in the console

```
`Listened to event test with payload: { something: 'some value' }`
```

Co-authored by @remidej @Feranchz
